test/functional/test_framework/util.py

```diff
@@ -47,20 +47,44 @@ def assert_fee_amount(fee, tx_size, feerate_BTC_kvB):
         raise AssertionError("Fee of %s BTC too high! (Should be %s BTC)" % (str(fee), str(target_fee)))
 
 
-def assert_equal(thing1, thing2, *args):
+def assert_equal(thing1, thing2, *args, err_msg=None):
```

Comment:
I preferer to use `err_msg` because in the future you are free to add the message if you need it for other stuff, I like to leave the port open to extension. 

What do you think?

Reply:
Sorry you're right, I misunderstood what `message` was doing in `assert_raises_message` - had a closer look now. Please disregard.